packages/lesswrong/components/tagging/AllWikiTagsPage.tsx

```diff
@@ -213,97 +207,29 @@ const uncategorizedRootTag = {
   slug: 'uncategorized-root',
   oldSlugs: [],
   description: {
+    _id: 'uncategorized-root',
     wordCount: 0,
-    // _id: 'uncategorized-root',
-    // html: '',
   },
   postCount: 0,
+  baseScore: 0,
   coreTagId: null,
   parentTagId: null,
   isArbitalImport: false,
 };
 
 // TODO: we really need to figure out a better way to handle this than slugs, especially with the merged rationality page
 const prioritySlugs = [
-  'rationality-1', 'ai', 'world-modeling', 
+  'rationality', 'rationality-1', 'ai', 'world-modeling', 
   'world-optimization', 'practical', 'community', 'site-meta'
 ] as const;
 
-// Define the buildTree function here
-function buildTree(
-  items: (AllTagsPageCacheFragment & { parentTagId: string | null })[],
-  _id: string | null = null, 
-  depth = 0, 
-  seen: Set<string> = new Set()
-): WikiTagNode[] {
-  if (depth > 5) {
-    // eslint-disable-next-line no-console
-    console.warn('Maximum depth exceeded in buildTree');
-    return [];
-  }
-
-  const filteredItems = items.filter(item => item.parentTagId === _id);
-  
-  // Add baseScore to all items first
-  const itemsWithScore = filteredItems.map(item => ({
-    ...item,
-    baseScore: generateBaseScore((item.description?.wordCount ?? 0) * 6, item.postCount)
-  }));
-
-  // Sort items by baseScore before processing
-  const sortedItems = itemsWithScore.sort((a, b) => b.baseScore - a.baseScore);
-
-  return sortedItems.flatMap(item => {
-    if (seen.has(item._id)) {
-      // eslint-disable-next-line no-console
-      console.warn(`Circular reference detected for item ${item.name} (${item._id})`);
-      return [];
-    }
-
-    seen.add(item._id);
-
-    const children = buildTree(items, item._id, depth + 1, new Set(seen)).sort((a, b) => b.baseScore - a.baseScore);
-
-    return [{ ...item, children }];
-  });
-}
-
-// Function to filter the tree based on tag IDs, but skip first level filtering
-function filterTreeByTagIds(
-  tree: WikiTagNode[], 
-  tagIds: string[], 
-  depth = 0
-): WikiTagNode[] {
-  return tree
-    .map(node => {
-      // At depth 0 (first level), don't filter out any nodes
-      const hasHit = depth === 0 ? true : tagIds.includes(node._id);
-      
-      let children: WikiTagNode[] = [];
-      if (node.children) {
-        // Always pass depth + 1 to children
-        children = filterTreeByTagIds(node.children, tagIds, depth + 1);
-      }
-      
-      if (hasHit || children.length > 0) {
-        return { ...node, children };
-      }
-      return null;
-    })
-    .filter(node => node !== null) as WikiTagNode[];
-}
-
-function isMatchingCoreTag(tag: AllTagsPageCacheFragment, slug: string): boolean {
-  return tag.core && (slug === tag.slug || tag.oldSlugs?.includes(slug));
-}
-
-const allTagsQuery = gql`
-  query AllTagsQuery {
-    AllTags {
-      ...AllTagsPageCacheFragment
+const tagsBySlugsQuery = gql`
```

Comment:
Surely this can be a useMulti with a basic view?

Reply:
We already have a `tagBySlug` view, this would just be a minor modification

Reply:
I thought about this for a half a moment and for no good reason decided that wouldn't work...